ethicml/data/csvs/utility.py

```diff
@@ -0,0 +1,8 @@
+"""Functions used in either generating data, or pre-processing raw data."""
+
+import numpy as np
+
+
+def sigmoid(x: np.ndarray) -> np.ndarray:
+    """As there is no np.sigmoid."""
+    return 1 / (1 + np.exp(-x))
```

Comment:
If you wanna be fancy and do it numerically stable:

```suggestion
    return np.where(x >= 0,  1 / (1 + np.exp(-x)),  np.exp(x) / (np.exp(x) + 1))
```

Reply:
ohohoho